BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditItemView.swift

```diff
@@ -65,6 +65,20 @@ struct AddEditItemView: View {
                         .accessibilityIdentifier("PersonalOwnershipPolicyLabel")
                 }
 
+                if case .add = store.state.configuration, store.state.type == .login,
+                   store.state.showLearnNewLoginActionCard {
```

Comment:
🤔 Could we wrap this logic directly in a computed property in the `state` so the view is as simple as possible? Furthermore, by doing so we can have quicker and easier to maintain tests.